src/holochainClient.js

```diff
@@ -21,16 +21,12 @@ export const MOCK_INDIVIDUAL_DNAS = {
 export const HOLOCHAIN_LOGGING = true && process.env.NODE_ENV !== 'test'
 let holochainClient
 
-const agentId = process.env.REACT_APP_AGENT_ID
-
-export function conductorInstanceId (instanceId) {
-  const formatInstanceId = instanceId => agentId + '::<happ_id>-' + instanceId
-
+export function conductorInstanceIdbyDnaAlias (instanceId) {
   return {
-    hylo: formatInstanceId('hylo'),
-    'happ-store': formatInstanceId('happ-store'),
-    hha: formatInstanceId('holo-hosting-app'),
-    holofuel: formatInstanceId('holofuel')
+    hylo: 'hylo',
+    'happ-store': 'happ-store',
+    hha: 'holo-hosting-app',
```

Comment:
This dictionary lookup only exists because of the hha to holo-hosting-app conversion.  It would be nice to either update nix to expect the 'hha' alias, or to update every appearance of 'hha' to 'holo-hosting-app' in hp-admin.  Happy to hear votes for either direction. For now, this works.